charts/policy-controller/README.md.gotmpl

```diff
@@ -0,0 +1,115 @@
+{{ template "chart.header" . }}
```

Comment:
Q: Why do we need this file ?

Reply:
You don't have to do `helm-docs --dry-run` then modify the README.md each time.   Instead you run `helm-docs` and it generates the entire file for you.

Reply:
Also makes it easy to validate because if `helm-docs` generates a diff, then you know it wasn't properly updated.

Reply:
https://github.com/norwoodj/helm-docs#markdown-rendering

Reply:
Sgtm, what do you think @cpanato ?

Reply:
As mentioned, let's revert this change and keep only the opt-in functionality.